module/VuFind/src/VuFind/View/Helper/Root/Flashmessages.php

```diff
@@ -78,7 +78,7 @@ protected function getClassForNamespace($ns)
     public function __invoke()
     {
         $html = '';
-        $namespaces = ['error', 'info', 'success'];
+        $namespaces = ['default', 'success', 'warning', 'error', 'info'];
         foreach ($namespaces as $ns) {
             $messages = array_merge(
                 $this->fm->getMessages($ns), $this->fm->getCurrentMessages($ns)
```

Comment:
It looks like the messages will appear in that order if there are multiple. I don't know any cases where that would happen but is this the order we'd want them to appear in?

Reply:
Good point, @crhallberg! I would probably list these in order of severity, so maybe `['error', 'warning', 'info', 'success', 'default']` would be better.

It might also make sense to move this from a local `$namespaces` variable to a `$this->namespaces` class property, so it would be easier to reorder or otherwise change in a subclass without rewriting code.

Reply:
Ok, I will reorder them. This order is the same as in the FlashMessenger class but it seems reasonable to stick to the 'severity' scale.